lib/cylc/rundb.py

```diff
@@ -171,14 +163,15 @@ class CylcSuiteDAO(object):
     CONN_TIMEOUT = 0.2
     DB_FILE_BASE_NAME = "cylc-suite.db"
     MAX_TRIES = 100
-    TABLE_BROADCASTS = "broadcasts"
+    TABLE_BROADCAST_EVENTS = "broadcast_events"
+    TABLE_BROADCAST_STATES = "broadcast_states"
     TABLE_TASK_JOBS = "task_jobs"
     TABLE_TASK_JOB_LOGS = "task_job_logs"
     TABLE_TASK_EVENTS = "task_events"
     TABLE_TASK_STATES = "task_states"
 
     TABLES_ATTRS = {
-        TABLE_BROADCASTS: [
+        TABLE_BROADCAST_EVENTS: [
```

Comment:
Given this change, we should have something in the restart code that migrates the old table to the new one so as to be able to upgrade suite versions.